### ISSUE TITLE
Failed to load the todo-manager package

### ISSUE THREAD

**Issue by pierluc-codes** [benjaminRomano/todo-manager#1]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.3
**System**: Linux Mint
**Thrown From**: [todo-manager](https://github.com/benjaminRomano/todo-manager) package, v0.1.0
### Stack Trace

Failed to load the todo-manager package

```
At Cannot find module './RegexMatcherUtil'

Error: Cannot find module './RegexMatcherUtil'
  at Module._resolveFilename (module.js:328:15)
  at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:270:25)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/home/plcstpierre/.atom/packages/todo-manager/lib/main.coffee:2:20)
  at Object.<anonymous> (/home/plcstpierre/.atom/packages/todo-manager/lib/main.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app.asar/src/package.js:715:34)
  at /usr/share/atom/resources/app.asar/src/package.js:190:28
  at Package.module.exports.Package.measure (/usr/share/atom/resources/app.asar/src/package.js:167:15)
  at Package.module.exports.Package.load (/usr/share/atom/resources/app.asar/src/package.js:181:12)
  at PackageManager.module.exports.PackageManager.loadPackage (/usr/share/atom/resources/app.asar/src/package-manager.js:373:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:452:30)
  at /usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:370:29
  at exit (/usr/share/atom/resources/app.asar/node_modules/settings-view/lib/package-manager.js:59:16)
  at triggerExitCallback (/usr/share/atom/resources/app.asar/src/buffered-process.js:213:47)
  at /usr/share/atom/resources/app.asar/src/buffered-process.js:220:18
  at Socket.<anonymous> (/usr/share/atom/resources/app.asar/src/buffered-process.js:98:18)
  at emitOne (events.js:82:20)
  at Socket.emit (events.js:166:7)
  at Pipe.close (net.js:464:12)

```
### Commands

```
     -2:10.6.0 editor:indent (atom-text-editor.editor.is-focused)
  2x -2:10.3.0 core:move-up (atom-text-editor.editor.is-focused)
     -2:09.9.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -2:09.9.0 snippets:expand (atom-text-editor.editor.is-focused)
     -2:09.9.0 editor:indent (atom-text-editor.editor.is-focused)
  2x -2:09.3.0 core:move-down (atom-text-editor.editor.is-focused)
     -2:08.9.0 core:move-up (atom-text-editor.editor.is-focused)
     -2:08.5.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
  2x -2:07.9.0 core:move-up (atom-text-editor.editor.is-focused)
     -2:07.4.0 core:move-down (atom-text-editor.editor.is-focused)
  5x -1:58.8.0 core:backspace (atom-text-editor.editor.is-focused)
  3x -1:56.9.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:52.6.0 core:move-up (atom-text-editor.editor.is-focused)
     -1:49.7.0 editor:newline (atom-text-editor.editor.is-focused)
  2x -1:46.3.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -1:18.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
jshint, v1.3.10
todo-manager, v0.1.0

# Dev
No dev packages
```

**Comment by benjaminRomano** [comment]
I think I fixed the problem. Can you update to v0.1.1 and verify for me?

**Comment by benjaminRomano** [comment]
Merged PR #2, and published fixes to apm. Todo-manager version with fixes is v0.1.2.
